var/spack/repos/builtin/packages/grib-api/package.py

```diff
@@ -9,9 +9,11 @@
 class GribApi(CMakePackage):
     """The ECMWF GRIB API is an application program interface accessible from
        C, FORTRAN and Python programs developed for encoding and decoding WMO
-       FM-92 GRIB edition 1 and edition 2 messages."""
+       FM-92 GRIB edition 1 and edition 2 messages. The API was deprecated
```

Comment:
For these deprecated packages, should we add `deprecated=True` to each version?

@skosukhin

Reply:
Sure! Let me know which versions (or all)